doc/scripts/genrest/genrest.py

```diff
@@ -90,4 +99,13 @@ def print_items(items, outdir, indent):
 f.write(".. list-table:: Configuration Options\n\n")
 conf = kconfiglib.Config(sys.argv[1])
 print_items(conf.get_top_level_items(), sys.argv[2],  0)
+
+# print_items creates separate .rst files for each configuration option as
+# well as a dictionary index of all these options. Print out the accumulated
+# config symbols index in alphabetic order
+
+sortedItems = sorted(itemIndex.keys())
```

Comment:
compact to 

`for item in sorted(item_index):`

item_index, when iterated, evaluates to the keys, by default, so no need to call keys()